CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # CHANGELOG
 
+## 0.38.0
+
+* The `Kafka.KafkaStatus.ListenerStatus.type` property has been deprecated for a long time, and now we do not use it anymore.
+  The current plan is to completely remove this property in the next schema version.
+  If needed, you can use the `Kafka.KafkaStatus.ListenerStatus.name` property, which has the same value.
```

Comment:
Can you add this to the `### Changes, deprecations and removals` section as we have for other releases? That way it will be listed together with the other notabale changes like this.